Zend/zend_objects_API.c

```diff
@@ -203,7 +203,7 @@ ZEND_API void ZEND_FASTCALL zend_objects_store_del(zend_object *object) /* {{{ *
 
 ZEND_API ZEND_COLD zend_property_info *zend_get_property_info_for_slot_slow(zend_object *obj, zval *slot)
 {
-	uintptr_t offset = (uintptr_t)slot - (uintptr_t)obj->properties_table;
+	uintptr_t offset = (uintptr_t)slot - (uintptr_t)obj;
```

Comment:
Perhaps this should be a macro in zend_compile.h ? e.g. somewhere here: https://github.com/php/php-src/blob/bd4333447e0dfbac097f60e0ed675277219dd743/Zend/zend_compile.h#L462-L469

I think it's better to place all those implementation details in a single place

Reply:
Can do. `OBJ_PROP_SLOT_TO_OFFSET()`?

Reply:
sURE